framework/src/userobject/ElementSubdomainModifier.C

```diff
@@ -189,12 +217,22 @@ ElementSubdomainModifier::updateBoundaryInfo(MooseMesh & mesh,
     for (auto side : elem->side_index_range())
     {
       const Elem * neighbor = elem->neighbor_ptr(side);
+      if (_complement_moving_boundary_specified)
```

Comment:
This code seems like it should go down in the block starting in line 238, where a similar operation is done on the moving boundary (and where's some extra code for parallel cases).